schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/spi/dm/JasUserSubscription.java

```diff
@@ -0,0 +1,56 @@
+package com.jasify.schedule.appengine.spi.dm;
+
+import com.jasify.schedule.appengine.dao.common.ActivityDao;
+import com.jasify.schedule.appengine.model.EntityNotFoundException;
+import com.jasify.schedule.appengine.model.activity.Activity;
+import com.jasify.schedule.appengine.model.activity.Subscription;
+import com.jasify.schedule.appengine.spi.transform.JasActivityTransformer;
+import com.jasify.schedule.appengine.spi.transform.JasSubscriptionTransformer;
+
+/**
+ * @author wszarmach
+ * @since 03/09/15.
+ */
+public class JasUserSubscription implements JasEndpointEntity {
+    private JasActivity activity;
+    private JasSubscription subscription;
+
```

Comment:
These should be
~~~java
  private Activity activity;
  private Subscription subscription;
~~~

the get transformed by the google endpoints api